### PR TITLE
Fix/monitoring script integration

### PR DESCRIPTION
**Pull request by ranjan-stha**

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -283,18 +309,16 @@ def __str__(self):
 
 class TagWisePerfMetrics(models.Model):
     tags = models.CharField(max_length=250, blank=True)
-    precision = models.FloatField(blank=True, null=True)
-    recall = models.FloatField(blank=True, null=True)
-    f1score = models.FloatField(blank=True, null=True)
-    support = models.IntegerField(blank=True, null=True)
+    scores = models.FloatField(blank=True, null=True)
```
Comment (k9845) [inline core/models.py]: lets name `score`, `scores` sounds to store many values.

File: core/models.py [hunk 2]
```diff
@@ -283,18 +309,16 @@ def __str__(self):
...
+    metrics = models.CharField(max_length=250, blank=True, null=True)
```
Comment (k9845) [inline core/models.py]: Same as above

**Review by k9845 — CHANGES_REQUESTED** [review]
Also, try to make fewer migrations.

File: core/models.py (hunk 2 above)
Comment (ranjan-stha) [reply core/models.py]: done

File: core/models.py (hunk 1 above)
Comment (ranjan-stha) [reply core/models.py]: done.

**Review by k9845 — APPROVED** [review]
LGTM !

File: nlp_scripts/model_monitoring/utils.py [hunk 3]
```diff
@@ -50,25 +50,37 @@ def get_model_info(
...
+    tags_lst = []
+    if isinstance(tags_collection, dict):
+        tags_items = [list(tags_collection.keys())]
+    elif isinstance(tags_collection, list):
+        if isinstance(tags_collection[0], dict):
+            tags_items = [list(i.keys()) for i in tags_collection]
+        else:
+            tags_items = tags_collection
+    else:
+        return [[]]
```
Comment (Rup-Narayan-Rajbanshi) [inline nlp_scripts/model_monitoring/utils.py]: Why [[]] ? Can't we return [] ?

**Review by Rup-Narayan-Rajbanshi — CHANGES_REQUESTED** [review]

File: nlp_scripts/model_monitoring/utils.py [hunk 4]
```diff
@@ -50,25 +50,37 @@ def get_model_info(
     return model_info, err_msg
 
 
-def group_tags(tags_collection: Dict) -> Dict[str, List[str]]:
+def group_tags(tags_collection: Union[Dict, List]) -> Dict[str, List[str]]:
```
Comment (Rup-Narayan-Rajbanshi) [inline nlp_scripts/model_monitoring/utils.py]: Return type mentioned here is diffrent than the return value.

File: nlp_scripts/model_monitoring/utils.py (hunk 3 above)
Comment (ranjan-stha) [reply nlp_scripts/model_monitoring/utils.py]: This is fixed.

**Review by Rup-Narayan-Rajbanshi — APPROVED** [review]

File: nlp_scripts/model_monitoring/utils.py (hunk 4 above)
Comment (ranjan-stha) [reply nlp_scripts/model_monitoring/utils.py]: fixed

**Review by k9845 — APPROVED** [review]

**Pull request merged**